src/dynamo-streams.ts

```diff
@@ -69,6 +77,7 @@ export type DynamoStreamHandlerHarnessConfig<Context> = {
 
 export type DynamoStreamHandlerHarnessContext<Entity> = {
   sendEvent: (event: TestEvent<Entity>) => Promise<void>;
+  obfuscateEvent: (event: DynamoDBStreamEvent) => DynamoDBStreamEvent;
```

Comment:
This change seems odd to me. It seems like this utility would not be valuable to external consumers.

It seems like it's been added to support the new test cases in `dynamo-streams.test.ts` -- does that sound right?

If so, could we use different assertions in that test case to avoid needing to modify this public API?

For example, perhaps we could assert on `logger.child`:

(pseudocode)
```typescript
const obfuscateKeys: ['apiSecret']

const testRecord = {
  someOtherValue: 'test-value',
  apiSecret: 'sensitive-value'
};

await sendEvent(testRecord);

expect(logger.child).toHaveBeenCalledWith({
  record: {
    someOtherValue: 'test-value',
    apiSecret: '[obfuscated]'
  }
})


```

Reply:
Will look. I guess I didn't understand the purpose of DynamoStreamHandlerHarnessContext, thought its entire purpose of existence was for the tests.

Reply:
Done. Way better. Great suggestion, thank you.